File: Problem 2/js solution/fibo.js
let previousNumber = 0
let currentNumber = 1
let nextNumber = 1
let sum = 0

while (nextNumber <= 4000000) {
  if (nextNumber % 2 === 0) {
    sum += nextNumber
  }
  nextNumber = currentNumber + previousNumber
  previousNumber = currentNumber
  currentNumber = nextNumber
}

console.log(sum)

File: Problem 2/python solution/fibo.py
previousNumber, currentNumber = 0, 1
nextNumber = 1
sum = 0

while nextNumber <= 4000000:
  if nextNumber % 2 == 0:
    sum += nextNumber
  nextNumber = currentNumber + previousNumber
  previousNumber, currentNumber = currentNumber, nextNumber

print(sum)